src/include/ndpi_api.h.in

```diff
@@ -1011,8 +1011,7 @@ extern "C" {
 		     u_int32_t src_v4, u_int32_t dst_v4,
 		     struct ndpi_in6_addr *src_v6, struct ndpi_in6_addr *dst_v6,
 		     u_int16_t src_port, u_int16_t dst_port,
-		     ndpi_protocol l7_protocol,
-		     ndpi_serializer *serializer);
```

Comment:
Isn't that parameter the most important? Otherwise I do not know, why I should even consider using this function.